delivery/repositories/staging_repository.py

```diff
@@ -1,72 +0,0 @@
-
-import os
-
-from sqlalchemy.orm.exc import NoResultFound
-
-from delivery.models.db_models import StagingOrder
-
-
-class DatabaseBasedStagingRepository(object):
-    """
-    A repository of staging orders backed by a database. It is able to create and commit new staging orders
-    to the database, and fetch them based on different factors.
-    """
-
-    def __init__(self, session_factory):
-        """
-        Instantiate a new DatabaseBasedStagingRepository
-        :param session_factory: factory method which can produce new sqlalchemy Session objects
-        """
-        self.session = session_factory()
-
-    def get_staging_order_by_source(self, source):
-        """
-        Get all staging orders based on their source
-        :param source: to search for
-        :return: All staging orders of that source as a list
-        """
-        return self.session.query(StagingOrder).filter(StagingOrder.source == source).all()
-
-    def get_staging_order_by_id(self, identifier, custom_session=None):
-        """
-        Get a staging order by id
-        NOTE
-        The custom_session used here is used because in the `StagingService` it is necessary to
```

Comment:
Might be an idea to create a ticket on looking further into this later, and then the note, if it doesn't exist.